common-arcana.lic

```diff
@@ -74,17 +78,11 @@ module DRCA
       .each do |name|
         activate_barb_buff?(name)
         waitrt?
-        pause settings.meditation_pause_timer.to_i if get_data('spells').barb_abilities[name]['type'].eql? 'meditation'
+        pause settings.meditation_pause_timer.to_i if get_data('spells').barb_abilities[name]['type'].eql?('meditation')
         DRC.fix_standing
       end
   end
 
-  # Deprecated, will be removed in a future update
-  # Use 'activate_babr_buff?' instead.
-  def activate_barb_buff(name)
-    activate_barb_buff?(name)
-  end
-
```

Comment:
Removing obsolete code now that all uses of this have been updated to use the new predicate `activate_barb_buff?`